modules/reactor/src/meshgenerators/ReactorMeshParams.C

```diff
@@ -79,6 +82,11 @@ ReactorMeshParams::ReactorMeshParams(const InputParameters & parameters)
   this->declareMeshProperty(RGMB::assembly_pitch, _assembly_pitch);
   this->declareMeshProperty("name_id_map", _name_id_map);
 
+  // Option to bypass mesh generation depends on value of Mesh/data_only
+  const auto & moose_mesh = _app.actionWarehouse().getMesh();
```

Comment:
I'll make this option a bit more accessible

Reply:
@loganharbour do you plan to have a function in mesh generator system to expose this data? I would still need to get the value of data_driven_generator from Reactor module